iOS Example Frame/iOS Example Frame/MainViewController.swift

```diff
@@ -42,20 +42,23 @@ class MainViewController: UIViewController, CardViewControllerDelegate {
     
     @IBAction func onClickGoTokenWithApplePay(_ sender: Any) {
         
-        // Step2 : Load ApplePay Payment Data
-        let header = Header()
-        let paymentData = PaymentData(header: header)
-    
-        let appleHeader = ApplePayTokenDataHeader(ephemeralPublicKey: paymentData.header.ephemeralPublicKey, publicKeyHash: paymentData.header.publicKeyHash, transactionId: paymentData.header.transactionId)
-        let applePayToken = ApplePayTokenData(version: paymentData.version, data: paymentData.data, signature: paymentData.signature, header: appleHeader)
-        
+        // Use example Apple Pay payment data.
+        guard let paymentDataURL = Bundle.main.url(
```

Comment:
have you tested that this works with Cocoapods, SPM and Carthage? Bundles related sections are most likely to differ between package managers.

Reply:
No, this example project is CocoaPods only.

Reply:
Ah whoops, didn't see the folder.